public/pages/DetectorResults/components/OutOfRangeModal/OutOfRangeModal.tsx

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Added the OutOfRangeModal code to this PR branch, for the sake of seeing the changes reflected here. The original modal was approved in PR 4 (#15). The only change is adding an additional `lastEnabledTime` param, and adding the param value in the modal text body. Screenshot added to this PR's description.

Reply:
thanks for the tip and screenshot.  Pretty helpful.